### ISSUE TITLE
webmin doesent show up on apache reverse proxy

### ISSUE THREAD

**Issue by camatthew** [webmin/webmin#899]
the reverse prosy doesent show up. I followed the guide

**Comment by qooob** [comment]
What do you mean _it doesn't show up_?

**Comment by camatthew** [comment]
when i acess the webmin folder i get 404

**Comment by qooob** [comment]
Shorthand for you.


_PROXY SETUP IN A SUB-DIRECTORY_
(http://www.webmin.com/apache.html)

**/etc/webmin/config**
  webprefix=/subdirname
  webprefixnoredir=1
  referers_none=1
  referer=192.168.122.2

**/etc/webmin/miniserv.conf**
  cookiepath=/subdirname

After editing those files restart Webmin by running `/etc/webmin/restart`

Next, is Apache set up:

For example, simplest way to do it on stock Apache setup is to edit `/etc/httpd/conf.d/ssl.conf` (for RHEL systems, with installed _mod_ssl_) and add before closing `</VirtualHost>`:


```
<IfModule mod_proxy.c>
	RewriteEngine On
        ProxyRequests Off
        SSLProxyEngine On
        SSLProxyVerify None 
	SSLProxyCheckPeerCN Off
	SSLProxyCheckPeerName Off
	SSLProxyCheckPeerExpire Off
              
        RewriteRule ^/subdirname$ /subdirname/ [R]
        ProxyPass "/subdirname" "https://192.168.122.2:10000"
        ProxyPassReverse "/subdirname" "https://192.168.122.2:10000"
        
        <Location /subdirname>
            Header always set X-Frame-Options SAMEORIGIN
	</Location>        
</IfModule>
```

Restart Apache after editing config file and you're done.

**Comment by camatthew** [comment]
what is subdirname

**Comment by qooob** [comment]
Follow my example and it will all work. _subdirname_ is an example, your full URL would look like:

```
http://mydomainname.com/subdirname
```

You can change _subdirname_ to anything you like.

**Comment by camatthew** [comment]
thanks

**Comment by camatthew** [comment]
should it be a fresh install

**Comment by camatthew** [comment]
can you answer me

**Comment by qooob** [comment]
It can be any install. On fresh install (only fresh install for Apache I'm talking about, not the system) it would work just like described above. I don't say it wouldn't work on non-fresh install, it could be slightly different though.

**Comment by qooob** [comment]
Yes, make sure you access your Webmin via Apache using `https`, as the described method for SSL only. For non-SSL it's close, I bet you could find this out.

**Comment by camatthew** [comment]
bash: cd: /etc/httpd: No such file or directory

**Comment by qooob** [comment]
Try `/etc/apache2` and make sure that it's installed at the first place.

**Comment by camatthew** [comment]
where in that fodler

**Comment by camatthew** [comment]
still wont work

**Comment by camatthew** [comment]
404 error

**Comment by camatthew** [comment]
redirects to default page

**Comment by qooob** [comment]
You need to share your configuration completely and how do you access your Webmin installation via proxy? Saying _still wont work_ is not going to help in solving your issue.

Do you have _mod_proxy_ installed?

**Comment by camatthew** [comment]
yes

**Comment by camatthew** [comment]
i use dataplicity

**Comment by iliaross** [comment]
❗️ If the issue isn't resolved, feel free to reopen it with updated details.
